docs/section4/4-11.md

```diff
@@ -0,0 +1,33 @@
+# Airside systems – Rule 4-11
+
+**Rule ID:** 4-11  
+ 
+**Rule Description:** Fan Schedules may be allowed to differ when Section G3.1.1(c) applies, i.e. baseline HVAC system type 3 or 4 has been used for spaces (zones) that differ by more than 40 equivalent full load hrs/week from other spaces (zones) served by the system.  
+
+**Rule Assertion:** P-RMR = B-RMR                                           
+**Appendix G Section:** Table G3.1 Section 4 Baseline Exception 3  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:** 
+
+1. Applies when baseline HVAC system type 3 or 4 has been used for spaces (zones) that differ by more than 40 equivalent full load hrs/week from other spaces (zones) served by the system.  
+
+**Function Call:** 
+
+1. get_zones_G3.1.1c_40_EFLHs()
+2. match_data_element()
+
+**Manual Check:** Check that fan schedules have been modeled appropriately in the baseline and proposed when fan schedules are allowed to differ between the baseline and proposed per Appendix G Section G3.1.1(c).
+
+**Applicability Check 1:** 
+- Get dict of zone id keys and hvac id values applicable to this check: `section_G3.1.1c_applies_zones_hvac_dict_b = get_zones_G3.1.1c_40_EFLHs(B_RMR)`
+- Create list of baseline hvac systems in which Appendix G Section G3.1.1(c) applies: `hvac_systems_section_G3.1.1c_applies_list_b = list(section_G3.1.1c_applies_zones_hvac_dict_b.values())`
+- For each hvac_b in the list in which Section G3.1.1c applies: `for hvac_b in hvac_systems_section_G3.1.1c_applies_list_b:`    
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system, outcome is undetermined: `UNDETERMINED`
+    - Case 2: Else, for each non applicable zone: `Else: IN_APPLICABLE`
+
```

Comment:
Perhaps the following logic should be used for the RDS:
1. Applicability check: verify that the baseline has (a) systems 5-9 (there are additional systems listed in G3.1.1(c) but it was a mistake that was corrected in addendum ab) AND (b) that there are baseline systems 3-4 on the same floor (i.e., zones served by these systems have the same zone.floor_name).   
2. Identify baseline HVAC systems associated with zones 1(b).
3. Identify proposed HVAC systems associated with zones 1(b). For each such system,
    if proposed HVAC system is multizone, and the baseline FanSystem.operating_schedule != TBD then pass, else fail.
4. The question is how to determine the correct schedule that should be modeled based on occupancy schedule of the associated spaces.

Reply:
@mkarpman I think i understand this logic. 
Though not sure why in (3) the proposed needs to be MZ. Even if the proposed is SZ, the baseline would need to be SZ system 3 or 4 depending on EFLH, correct? Do we need (3) at all?

Reply:
I believe this RDS is getting quite complicated, it will be difficult to implement and difficult to verify. My suggestion would be to only check for Case 1 - hvac schedules match and the proposed and baseline are both served by either SZ  or MZ systems. All other cases could be 'UNDETERMINED'.

Not sure if there is a clear fail case as well, if yes, we could add that. I think each fail case could have an exception which  could be causing it so maybe there is no value in testing all those additional scenarios as well.

Reply:
Other discussions:

![image](https://user-images.githubusercontent.com/8323384/162830701-6849fc5b-a3d1-482d-8429-2cfd85b13ee6.png)

Reply:
Exhaust fan schedule is still an open issue https://github.com/open229/ruleset-model-description-schema/issues/93 .

Reply:
Issue closed. Exhaust will be checked via an umbrella check on the zone level.